storage-mongodb/src/main/java/com/linagora/calendar/storage/mongodb/MongoConstants.java

```diff
@@ -0,0 +1,23 @@
+/********************************************************************
+ *  As a subpart of Twake Mail, this file is edited by Linagora.    *
+ *                                                                  *
+ *  https://twake-mail.com/                                         *
+ *  https://linagora.com                                            *
+ *                                                                  *
+ *  This file is subject to The Affero Gnu Public License           *
+ *  version 3.                                                      *
+ *                                                                  *
+ *  https://www.gnu.org/licenses/agpl-3.0.en.html                   *
+ *                                                                  *
+ *  This program is distributed in the hope that it will be         *
+ *  useful, but WITHOUT ANY WARRANTY; without even the implied      *
+ *  warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR         *
+ *  PURPOSE. See the GNU Affero General Public License for          *
+ *  more details.                                                   *
+ ********************************************************************/
+
+package com.linagora.calendar.storage.mongodb;
+
+public class MongoConstants {
+    public static final int MONGO_DUPLICATE_KEY_CODE = 11000;
```

Comment:
Separate commit for this?

Reply:
https://github.com/linagora/twake-calendar-side-service/pull/546